runtimes/react-native/adding-rive-to-expo.mdx

```diff
@@ -3,110 +3,115 @@ title: 'Adding Rive to Expo'
 description: 'Rive React Native Expo. '
 ---
 
-To enable your Expo application to run Rive animations you'll need to include the native Android and iOS libraries.
+To use Rive with Expo, you'll need to install the `rive-react-native` package.
 
-To achieve this you can make use of [development builds](https://docs.expo.dev/develop/development-builds/introduction/) or by [generating the native projects with prebuild](https://docs.expo.dev/workflow/customizing/#generate-native-projects-with-prebuild). A development build can be seen as your own version of the Expo Go client that includes custom native libraries. We suggest reading [Expo's documentation](https://docs.expo.dev/workflow/customizing/#generate-native-projects-with-prebuild) for more information.
+Because this package contains custom native code, it's not compatible with Expo Go. Instead, you'll need to use a development build, which gives you full access to native modules.
 
-### Setup
+> **Note:**
+> Development builds are the [recommended setup for production apps](https://github.com/expo/fyi/blob/main/expo-go-usage.md).
 
-This guide will demonstrate how you can make use of prebuild to generate the required native code.
+This guide will walk you through integrating Rive into your Expo project, including installing dependencies, configuring your build, and testing your animations.
 
-Start by creating a new expo app (or use an existing project):
+## Initial Setup
+
+If you don’t already have a project, create a new Expo app:
 
 ```bash
 npx create-expo-app MyRiveApp
 ```
 
-Add the Rive React Native package:
+Install the Expo development client:
 
 ```bash
-npx expo install rive-react-native
+npx expo install expo-dev-client
 ```
 
-Add a Rive animation to a view:
+Then install the Rive package:
 
-```javascript
-import Rive from 'rive-react-native';
-import { StyleSheet, Text, View } from 'react-native';
-
-function RiveDemo() {
-  return <Rive
-      url="https://public.rive.app/community/runtime-files/2195-4346-avatar-pack-use-case.riv"
-      artboardName="Avatar 1"
-      stateMachineName="avatar"
-      style={{width: 400, height: 400}}
-  />;
-}
+```bash
+npx expo install rive-react-native
+```
 
-export default function App() {
-  return (
-    <View style={styles.container}>
-      <RiveDemo />
-    </View>
-  );
-}
+## iOS Minimum Version
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: '#fff',
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-});
-```
+Rive for iOS requires a minimum deployment target of `14.0`.
```

Comment:
Since this is only relevant for older setups, can we throw it in an info tag, rather than including it as a step? I think lines 35 - 41, 47 - 63 with something like: 

```
<info>
If you're using Expo SDK 51 or earlier, update the minimum deployment target to `14.0`. 


If you're using CNG, you can set the minimum iOS deployment ...
</info>
```